.github/workflows/release.yaml

```diff
@@ -1,11 +1,7 @@
 name: Release
 on:
   workflow_dispatch:  # allows triggering a github action manually - see 'Actions' tab
```

Comment:
we should add 

```yaml
on:
  push:
    tags:
      - '*'
```

if we want to trigger this workflow when new tag event in GitHub

Reply:
We decided that we want to keep the manual trigger for now.